testGeogrid_ISCE.py

```diff
@@ -279,22 +243,47 @@ def runGeogrid(info, info1, dem, dhdx, dhdy, vx, vy, srx, sry, csminx, csminy, c
     obj.winssmname = "window_stable_surface_mask.tif"
     obj.winro2vxname = "window_rdr_off2vel_x_vec.tif"
     obj.winro2vyname = "window_rdr_off2vel_y_vec.tif"
-    
+
     obj.getIncidenceAngle()
     obj.geogrid()
 
-
-
-def runGeogridOptical(info, info1, dem, dhdx, dhdy, vx, vy, srx, sry, csminx, csminy, csmaxx, csmaxy, ssm, urlflag):
+    run_info = {
+        'chipsizex0': obj.chipSizeX0,
+        'vxname': vx,
+        'vyname': vy,
+        'sxname': kwargs.get('dhdxs'),
+        'syname': kwargs.get('dhdys'),
+        'maskname': kwargs.get('sp'),
+        'xoff': None,  # FIXME: Get from C object (is calculated) or another source
+        'yoff': None,  # FIXME: Get from C object (is calculated) or another source
+        'xcount': None,  # FIXME: Get from C object (is calculated) or another source
+        'ycount': None,  # FIXME: Get from C object (is calculated) or another source
+        'dt': obj.repeatTime,
+        'epsg': kwargs.get('epsg'),
+        'XPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'YPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'pixsizex': None,  # FIXME: Get from C object (is calculated) or another source
+        'rangePixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'azimuthPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
```

Comment:
Please remove these as they are not necessary. `rangePixelSize` is same as `XPixelSize` above and similar for y. `pixelsizex` is defined in testautoRIFT and is equivalent to `XPixelSize` as well. In other words, we should try to simplify runinfo definition and make it look the same between radar and optical although specific terms could have their own (and slightly different) meanings. 

I will fix the rest of the 'FIXME's.

Reply:
Done!